domains/df.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "df's Dev Website",
+    "repo": "https://github.com/DF-wu",
+    "owner": {
+        "username": "DF-wu",
+        "linkedin": "https://www.linkedin.com/in/chufei-wu-b33990164/",
+        "email": "df@dfder.tw"
+    },
+    "record": {
+        "CNAME": "profile.dfder.tw"
```

this domain redirects to https://df-wu.github.io/DF-wu/, you can use the CNAME as df-wu.github.io and use the is-a.dev domain in that repo (as u did before but u didnt send the website link)
```suggestion
        "CNAME": "df-wu.github.io"
```

---

oh, ur  reply is so fast ;)
yes, `https://df-wu.github.io/DF-wu/` is what I want to redirect to. 
But I don't know how it will be work. I will dig it out.
So I can leave the `CANME: df-wu.github.io` and works?

---

yes, you just need to register this domain in that repo's settings after its merged